integration-test/example.test.js

```diff
@@ -0,0 +1,85 @@
+import app from '../example/devServer.js'
+import { launch } from 'puppeteer'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+
+expect.extend({ toMatchImageSnapshot })
+
+const PORT = 9000
+
+const urlWhitelist = [
+  new RegExp(`http:\/\/localhost:${PORT}.*`),
+  /https:\/\/unpkg\.com\/react@[\d\.]+\/dist\/react\.min\.js/,
+  /https:\/\/unpkg\.com\/react-dom@[\d\.]+\/dist\/react-dom\.min\.js/,
+  /https:\/\/unpkg\.com\/react-dom@[\d\.]+\/dist\/react-dom-server\.min\.js/,
+  /https:\/\/cdnjs\.cloudflare\.com\/ajax\/libs\/babel-core\/[\d\.]+\/browser\.min\.js/
+]
+
+const globalCss = `
+  * {
+    -webkit-animation: unset !important;
+    animation: unset !important;
+    font-family: 'Arial' !important;
+    font-smooth: never !important;
+  }
+  pre, pre * {
+    font-family: 'Courier' !important;
+  }
+  .hero-header {
+    min-height: auto !important;
+  }
+`
+
+function startServer(app, port) {
+  return new Promise((resolve, reject) => {
+    const server = app.listen(port, error => {
+      if (error) {
+        reject(error)
+      }
+      resolve(server)
+    })
+  })
+}
+
+describe('example page', () => {
+  let server
+  let browser
+  let page
+
+  beforeAll(async () => {
+    server = await startServer(app, PORT)
+    browser = await launch({
+      executablePath: process.env.GOOGLE_CHROME_BINARY || undefined
+    })
+    browser = await launch({
+      executablePath: process.env.GOOGLE_CHROME_BINARY || undefined,
+      args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-lcd-text']
```

Comment:
Regarding your comment:

> `'--no-sandbox', '--disable-setuid-sandbox'`

These help with running in a `sudo` Travis environment. I'll make the appropriate changes in a follow-up PR to enable the test on Travis.

> `--disable-lcd-text`

This in combination with a forced `deviceScaleFactor` (can be `> 1` on some Retina/High-Res devices by default) prevents some diverging font rendering. Not perfect but close enough.

Reply:
Wow it's clear how much you struggled to come up w/ the combination 😄 💯

Reply:
Thanks!